docs/diagrams/ArchitectureSequenceDiagram.puml

```diff
@@ -8,13 +8,13 @@ Participant ":Logic" as logic LOGIC_COLOR
 Participant ":Model" as model MODEL_COLOR
 Participant ":Storage" as storage STORAGE_COLOR
 
-user -[USER_COLOR]> ui : "delete 1"
+user -[USER_COLOR]> ui : "addpatient  i/T1234567A  n/JOHN d/2000-01-01 p/ 12345678"
 activate ui UI_COLOR
 
-ui -[UI_COLOR]> logic : execute("delete 1")
+ui -[UI_COLOR]> logic : execute("addpatient  i/T1...")
 activate logic LOGIC_COLOR
 
-logic -[LOGIC_COLOR]> model : deletePerson(p)
+logic -[LOGIC_COLOR]> model : addPerson(p)
```

Comment:
we dont have an addPerson method I think, could u maybe change this to addPatient instead?